### DISCUSSION TITLE
channel change by itself

### DISCUSSION THREAD

**Discussion by argy85** [AndreyPavlenko/Fermata#223] [General]
it often changes channels on its own forward, anyone have this issue? I have an Xiaomi Redmi Note 10 Pro

**Comment by AndreyPavlenko** [comment]
If playback is failed, for some reason, it skips to the next channel.

**Comment by argy85** [comment]
ok, thanks, it would be nice to implement some kind of retry 3 times before
doing this skip forward since the buffering could be momentary

Il dom 5 feb 2023, 18:56 Andrey ***@***.***> ha scritto:

**Comment by Cristianpxx** [comment]
Mine is like that too rsr
